TicTacToe/TicTacToe/UI.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+
+namespace TicTacToe {
+
+    public class UI {
+
+        private Board board;
+        private IO io;
+
+        public UI(Board board, IO io) {
+            this.board = board; 
+            this.io = io;
+        }
+
+        public void NewGameView() {
+            string[] gameBoard = this.board.GameBoard; 
+            this.io.Print(Greeting());
+            this.io.Print(Instructions());
+            string board = GameBoard(gameBoard);
+            this.io.Print(BoardHeader(TurnPrompt("X")));
+            this.io.Print(BoardBorder(board));
+        }
+
+        public void PlaceMarker() {
+            string input = this.io.GetInput();
+            int move = Int32.Parse(input);
```

Comment:
Eventually, you'll want to consider how this can fail and handle the various cases gracefully.

Reply:
+1 to this